frontend/simple/js/backend/hapi.js

```diff
@@ -95,17 +88,28 @@ export class HapiBackend extends Backend {
     store.dispatch('saveSettings')
     return res
   }
+  // TODO add event strean method returning string.transform
+  async latestHash (contractId: string) {
+    let response = await fetch(`${process.env.API_URL}/latestHash/${contractId}`).then(r => r.json())
+    return response.data.hash
+  }
+  async eventsSince (contractId: string, since: string) {
+    let response = fetch(`${process.env.API_URL}/events/${contractId}/${since}`).then(r => r.json())
```

Comment:
This should have failed the linter check? (Did it not? If so there's another bug!) There must be an `await` in front of `fetch`